sessionctx/variable/sysvar.go

```diff
@@ -796,6 +796,17 @@ var defaultSysVars = []*SysVar{
 	}, GetGlobal: func(s *SessionVars) (string, error) {
 		return strconv.FormatFloat(PreparedPlanCacheMemoryGuardRatio.Load(), 'f', -1, 64), nil
 	}},
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBEnableGeneralPlanCache, Value: BoolToOnOff(DefTiDBEnableGeneralPlanCache), Type: TypeBool, SetSession: func(s *SessionVars, val string) error {
+		s.EnableGeneralPlanCache = TiDBOptOn(val)
+		return nil
+	}},
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBGeneralPlanCacheSize, Value: strconv.FormatUint(uint64(DefTiDBGeneralPlanCacheSize), 10), Type: TypeUnsigned, MinValue: 0, MaxValue: 100000, SetSession: func(s *SessionVars, val string) error {
```

Comment:
It seems the minValue for TiDBPrepPlanCacheSize is 1. See line 781. Should we keep them the same?

Reply:
Maybe we can change this in the next PR or maybe not changed. Both is OK for me.

Reply:
> Maybe we can change this in the next PR or maybe not changed. Both is OK for me.

Got it. Before, minValue = 0 is used to let the general plan cache OFF, I forget to revised back. I 'll fix it in next pr.